android/build.gradle

```diff
@@ -258,7 +256,7 @@ android {
         targetSdkVersion safeExtGet("targetSdkVersion", 30)
         versionCode 1
         versionName "1.0"
-        buildConfigField "boolean", "IS_NEW_ARCHITECTURE_ENABLED", IS_NEW_ARCHITECTURE_ENABLED.toString()
+        buildConfigField("boolean", "IS_NEW_ARCHITECTURE_ENABLED", IS_NEW_ARCHITECTURE_ENABLED.toString())
```

Comment:
I know this syntax is not really intuitive but we use it quite consistently here so probably there's no need to make this change.

Reply:
Actually a few lines below this syntax is used, so I made the change for consistency there.